### ISSUE TITLE
Missing Season/Episode Not Working

### ISSUE THREAD

**Issue by Yorik-Code** [Moussa/Transmogrify-for-Plex-firefox#5]
Using the latest plugin for Chrome and Firefox. I am unable to see the missing season and episodes in my TV Shows. I have diabling and reenabling the feature. It was working a few weeks ago. I am unsure of what I can do to test this further.

**Comment by pdille** [comment]
This should now be fixed in the latest release (1.4). Let us know if this isn't the case.

**Comment by rgraville** [comment]
Firefox add on site still only has 1.3.4.1 as of 23/SEP/2015.
Is there a way to download the 1.4 .xpi file directly?

**Comment by pdille** [comment]
If you go here, you can get the latest:

https://addons.mozilla.org/en-us/firefox/addon/transmogrify-for-plex/versions/

Looks like it isn't on the main page because the file isn't signed. Hmm. This is an issue because I believe the next version of FireFox will refuse to install non-signed addons.  @Moussa what does it take to get these files signed for use in FireFox?

**Comment by Moussa** [comment]
It's not signed and on the main page yet because it's still in the review queue.

**Comment by pdille** [comment]
Hmm, why is version 1.3.6 also not on the main page? That came out in June and 1.3.4.1 seems to be the last signed one.

**Comment by Moussa** [comment]
I don't believe 1.3.6 was fully reviewed for some reason, and never became an approved update.
